### PR TITLE
feat: redesign toasts

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Users seems to not get that the toast messages are not disappearing automatically. After some discussions w/ Mischa the first solution - to replace the ( x ) with more visible call-to-action.

The tooltip was removed and the message visible area was increased to be always 3 lines with a scroll if needed (aligned w/ Mischa)

# Tests

- Design update only, no tests needed.

# Screenshots

## warning
https://user-images.githubusercontent.com/98811342/163381000-8f3699f7-cb7a-4cf0-a3f6-d595c320ae0c.mov

## error
https://user-images.githubusercontent.com/98811342/163381020-18be7817-a74a-4419-9b15-a21c99f765ec.mov

## default
https://user-images.githubusercontent.com/98811342/163381036-0c7ba437-5a29-4a10-974a-94d40d2690bb.mov

### CODE REVIEW

File: frontend/svelte/src/lib/components/ui/Toast.svelte [hunk 1]
```diff
@@ -77,26 +86,25 @@
...
 
-  button {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
-
-    padding: 0;
+  button.close {
+    // rewrite default button styles
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/components/ui/Toast.svelte:103]: `//` isn't a valid CSS comment I think, not sure how it renders. You can use`/* */`.

**Review by peterpeterparker — APPROVED** [review]
Looks good and I like the solution, thanks a lot 👍

Please just have a look to my comment about the comment (🤪) before merge

File: frontend/svelte/src/lib/components/ui/Toast.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/svelte/src/lib/components/ui/Toast.svelte:103]: TID
image

Comment (peterpeterparker) [reply frontend/svelte/src/lib/components/ui/Toast.svelte:103]: No it's cool then, did not knew 🤷‍♂️. Thanks for the feedback

**Review by lmuntanerDfinity — APPROVED** [review]
I love the change. Is it too much to ask to add a success toast now? Or in another PR?

**Comment by mstrasinskis** [comment]
> I love the change. Is it too much to ask to add a success toast now? Or in another PR?

Let's do it in another PR ;)

**Pull request merged**